debian/control

```diff
@@ -916,6 +912,28 @@ Description: MySQL compatibility links to mariadb-server binaries/scripts.
  This package is also useful for users with strong  background in MySQL
  maintenance, but wants to switch to MariaDB
 
+Package: mariadb-server-galera
+Architecture: any
+Depends: galera-4 (>= 26.4),
+         gawk,
+         iproute2 [linux-any],
+         lsof [linux-any],
+         mariadb-server,
+         rsync,
+         socat,
+         ${misc:Depends},
+         ${perl:Depends},
```

Comment:
I didn't think anything here was perl.

Depends on mariadb-backup too?

Reply:
Perl is correct. I don't know mariadb-backup. I'll check it out

Reply:
mariadb-backup is one of the SST mechanisms, probably the one that should be used, but not quite enough to override the default.

Comments where intentional in the configuration file - not having things in there that are default. Odd this was in a client package.

Reply:
This should now be resolved if build goes well should check RPMs